luthier/config/src/tables.lua

```diff
@@ -136,16 +136,15 @@ M.physical = {
     Logical tables are defined in table M.logical. They map logical table
     names to their configuration:
 
-    * name - The name of the table
-        Defaults to the configuration's key
+    * key - The name of the logical table
     * description - Brief documentation about the logical table.
         Defaults to the configuration's key
     * metrics - A set of API metrics' name for this logical table, these 
         should be a subset of the metrics configured in metrics.lua.
     * dimensions - A set of dimensions for this logical table, the set of
         dimensions should be a subset of dimensions in its dependent physical
         tables.
-    * granularity - A group of available granularities of this logical table, the
+    * granularities - A group of available granularities of this logical table, the
         granularity can be "all", "hour", "day", "week", or "month".
     * physicalTables - A list of the names of the physical tables that this
         logical table depends on
```

Comment:
Missing documentation for `dateTimeZone`, and `type`. Make sure the documentation includes what the
default is, if any.

Reply:
Done.